app/locales/de.json

```diff
@@ -86,7 +86,7 @@
             "menubarIcon": "Symbol in Menüleiste:",
             "colour": "Farbig",
             "monochrome": "Einfarbig",
-            "invertedMonochrome": "Einfarbig Invertiert"
+            "invertedMonochrome": "Einfarbig Invertiert",
```

Comment:
one small thing :) you have extra `,` left after removing the unwanted strings.

Reply:
right I was too fast.